src/app/myworkflows/my-workflow/my-workflow.component.html

```diff
@@ -54,6 +54,20 @@
                         >
                           <mat-icon>add</mat-icon>
                         </button>
+                        <button mat-icon-button [matMenuTriggerFor]="myWorkflowsActions"><mat-icon>more_vert</mat-icon></button>
+                        <mat-menu #myWorkflowsActions="matMenu">
+                          <button
+                            mat-menu-item
+                            color="primary"
+                            type="button"
+                            (click)="addToExistingWorkflows()"
+                            [disabled]="isRefreshing$ | async"
+                            matTooltip="Discover workflows added by others using your linked source control accounts"
+                            matTooltipPosition="after"
+                          >
+                            <span>Find Existing Workflows</span>
```

Comment:
I prefer "Discover" over "Find", like you have in the tooltip.

I also think this is a little vague... where is it going to find them?

Discover Existing Dockstore Workflows?